test/sample-test/run_sample_test.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import io
+import json
+import tarfile
+import utils
+from datetime import datetime
+from kfp import Client
+
+###### Input/Output Instruction ######
+# input: yaml
+# output: local file path
+
+
+# Parsing the input arguments
+def parse_arguments():
+  """Parse command line arguments."""
+
+  parser = argparse.ArgumentParser()
+  parser.add_argument(
+      '--input',
+      type=str,
+      required=True,
+      help='The path of a pipeline package that will be submitted.')
+  parser.add_argument(
+      '--result',
+      type=str,
+      required=True,
+      help='The path of the test result that will be exported.')
+  parser.add_argument(
+      '--output', type=str, required=True, help='The path of the test output')
+  parser.add_argument('--testname', type=str, required=True, help='Test name')
+  parser.add_argument(
+      '--namespace',
+      type=str,
+      default='kubeflow',
+      help='namespace of the deployed pipeline system. Default: kubeflow')
+  args = parser.parse_args()
+  return args
+
+
+def main():
+  args = parse_arguments()
+  test_cases = []
+  test_name = args.testname + ' Sample Test'
+
+  ###### Initialization ######
+  host = 'ml-pipeline.%s.svc.cluster.local:8888' % args.namespace
+  client = Client(host=host)
+
+  ###### Check Input File ######
+  utils.add_junit_test(test_cases, 'input generated yaml file',
+                       os.path.exists(args.input), 'yaml file is not generated')
+  if not os.path.exists(args.input):
+    utils.write_junit_xml(test_name, args.result, test_cases)
+    print('Error: job not found.')
+    exit(1)
+
+  ###### Create Experiment ######
+  experiment_name = args.testname + ' sample experiment'
+  response = client.create_experiment(experiment_name)
+  experiment_id = response.id
+  utils.add_junit_test(test_cases, 'create experiment', True)
+
+  ###### Create Job ######
+  job_name = args.testname + '_sample'
+  ###### Test-specific parameters #######
+  if args.testname == 'tfx_cab_classification':
+    params = {
```

Comment:
Nit: These are "arguments", not "parameters".
These are parameters:
```python
def my_func(param1, param2):
```

These are arguments (for the parameters):
```python
my_func(param1=arg1, param2=arg2):
```

Reply:
Thanks for correcting!